UnitySetup/UnitySetup.psm1

```diff
@@ -888,6 +993,19 @@ function Request-UnitySetupInstaller {
     }
 }
 
+function StartProcessWithArgs($startProcessArgs)
+{
+    $process = Start-Process @startProcessArgs
```

Comment:
This process will write its output to the host I believe, which is likely not what we want. Might be good to pull out how we invoke unity in Start-UnityEditor to this method as it handles some of the flaws of `Start-Process`.

Reply:
I see that you pulled this from elsewhere so I'm ok leaving the output as is here.

Reply:
I think writing output is necessary during installation, because this may take a lot of time and it is probably good to be able to watch for its progress. Though I'll play with it for a bit to find out if there is something to improve.

Reply:
There is already `-Wait` in each `$startProcessArgs`, but yes, it would be better to have it here.

Reply:
I'd leave it as is, now that I see it aligns to the previous behavior. =)

Reply:
So I end up with removing this function since it is used in single place now. Previously I tried to re-use it, but it turned out to be unnecessary.